### PR TITLE
updated the server upgrade page refering to PR:2809

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
Signed-off-by: Dishank Tiwari <68427347+dishanktiwari2501@users.noreply.github.com>

### Description

The upgrade-server.md page has been updated with reference to the following PRS:
- [2834](https://github.com/chef/chef-web-docs/issues/2834)
- [2809](https://github.com/chef/chef-web-docs/issues/2809)

### Issues Resolved

The fixes were made in the chef-web-docs repo. The same changes have been made to the chef-server.

### Check List

- [] New functionality includes tests
- [x] All buildkite tests pass
- [x] Full omnibus build and tests in buildkite pass
- [] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: docs-chef-io/content/server/upgrade_server.md [hunk 1]
```diff
@@ -14,43 +14,37 @@ aliases = ["/upgrade_server.html", "/upgrade_server/"]
...
+
+## Upgrade Matrix
+
+If running a Chef Infra Server 12.17.15 or later you can upgrade directly to the latest releases of Chef Infra Server 14. If you are running a release prior to 12.17.15 you must perform a stepped upgrade as outlined below.
+
+| Running Version | Upgrade To Version | Requires License | Supported Version |
 |---------|---------|------|-----------|
 | 13 | 14 | Yes | Yes |
-| 12.17.15 | 14 | Yes | Yes |
+| 12.17.15 | 14 | Yes | No |
```
Comment (PrajaktaPurohit) [inline docs-chef-io/content/server/upgrade_server.md:30]: Des the Supported Version column indicate if the Running Version is supported?

File: docs-chef-io/content/server/upgrade_server.md [hunk 2]
```diff
@@ -14,43 +14,37 @@ aliases = ["/upgrade_server.html", "/upgrade_server/"]
...
 
-{{< warning >}}
-
-Do not upgrade your production server. First, upgrade in your test server, and then upgrade your production server.
-
-{{< /warning >}}
-
-Chef Infra Server 14 uses Elasticsearch as its search index.
+Chef Infra Server 14 moved from Solr to Elasticsearch as its search index.
 {{% server_upgrade_duration %}}
```
Comment (PrajaktaPurohit) [inline docs-chef-io/content/server/upgrade_server.md:43]: I see what server_upgrade_duration translates into in the rendered page; but cannot seem to find where it lives.

**Review by PrajaktaPurohit — APPROVED** [review]
Looks good to me! Left some questions inline. Looks like the PR will need a rebase before merging.

File: docs-chef-io/content/server/upgrade_server.md (hunk 2 above)
Comment (kagarmoe) [reply docs-chef-io/content/server/upgrade_server.md:43]: It is a shortcode (a partial) so it is in the main repo, which makes it available to all of the docs sets. We pull the partials in through the _vendor files, which can't be edited in this repo. 

The file is at: https://github.com/chef/chef-web-docs/blob/master/themes/docs-new/layouts/shortcodes/server_upgrade_duration.md

You can preview the content at https://deploy-preview-2208--chef-server.netlify.app/upgrade_server

Comment (kagarmoe) [reply docs-chef-io/content/server/upgrade_server.md:43]: @PrajaktaPurohit Do you have any concerns about this content, or can I resolve this issue?

File: docs-chef-io/content/server/upgrade_server.md (hunk 1 above)
Comment (kagarmoe) [reply docs-chef-io/content/server/upgrade_server.md:30]: I'm not sure if I'm following your meaning. The docs don't know what version of Chef Infra Server is running, although that would be a cool feature.

**Review by tas50 — APPROVED** [review]
Looks good from the product side of things

**Review by kagarmoe — APPROVED** [review]

**Review by PrajaktaPurohit — APPROVED** [review]

**Pull request merged**